docker/BUILD

```diff
@@ -71,7 +82,7 @@ debug_docker_build(
 )
 
 debug_docker_build(
-    entrypoint = ["/usr/local/bin/pilot"],
+    entrypoint = ["/usr/local/bin/agent"],
```

Comment:
Can we use istio-agent ? In raw VM (and in docker as well ) the agent name is to vague, there are many other agents.